test/python/compiler/test_transpiler.py

```diff
@@ -828,7 +829,7 @@ def test_initialize_FakeMelbourne(self):
         out_dag = circuit_to_dag(out)
         reset_nodes = out_dag.named_nodes("reset")
 
-        self.assertEqual(reset_nodes, [])
+        self.assertEqual(len(reset_nodes), 3)
```

Comment:
Should we extend this test to check all preset pass managers by looping over the possible optimization levels?

Reply:
I added such a test case.